packages/shared-state-network_nodes/files/usr/lib/lua/network-nodes.lua

```diff
@@ -0,0 +1,100 @@
+local shared_state = require("shared-state")
+local utils = require("lime.utils")
+local config = require("lime.config")
+
+local network_nodes = {}
+
+function network_nodes.node(hostname, member, fw_version, board, ipv4, ipv6)
+    return {hostname=hostname, member=member, fw_version=fw_version, board=board, ipv4=ipv4, ipv6=ipv6}
+end
+
+function network_nodes.serialize_for_network_nodes(node)
```

Comment:
same as for some of the following, these look like private methods belonging to the module.